website/src/client/components/Import/ImportFilesManager.tsx

```diff
@@ -21,7 +21,7 @@ type Props = {
   className?: string;
 };
 
-type FileItem = File | WebkitFileEntry | WebkitDirectoryEntry;
+type FileItem = File | WebkitFileEntry | WebkitDirectoryEntry | globalThis.FileSystemEntry;
```

Comment:
I don't think this one is required, right? The entries aren't `FileItem`s or `FileSystemEntry`'s, it's something that's being used after everything is processed.

Reply:
without this added type, there is a typescipt error [here](https://github.com/expo/snack/blob/ad8634a384f5f586af2ddf89effa19483dbc1cb3/website/src/client/components/Import/ImportFilesManager.tsx#L90). The array has both `File` and `FileSystemEntry` items.
<img width="844" alt="Screen Shot 2022-04-15 at 3 56 33 PM" src="https://user-images.githubusercontent.com/5482800/163650830-14b581eb-859b-4d59-94e7-edfdfec50039.png">